packages/react-core/src/demos/examples/AlertGroup/AlertGroupToastWithNotificationDrawer.tsx

```diff
@@ -0,0 +1,365 @@
+    }
+    return '';
+  };
+
+  const getUniqueId = () => new Date().getTime();
+
+  const getTimeCreated = () => {
+    const dateCreated = new Date();
+    return (
+      dateCreated.toDateString() +
+      ' at ' +
+      ('00' + dateCreated.getHours().toString()).slice(-2) +
+      ':' +
+      ('00' + dateCreated.getMinutes().toString()).slice(-2)
+    );
+  };
+
+  const removeAlert = (key: React.Key) => {
+    setAlerts(prevAlerts => prevAlerts.filter(alert => alert.props.id !== key.toString()));
+  };
+
+  const removeAllAlerts = () => {
+    setAlerts([]);
+  };
+
+  const onAlertGroupOverflowClick = () => {
+    removeAllAlerts();
+    setDrawerExpanded(true);
+  };
+
+  const onMaxDisplayedAlertsMinus = () => {
+    setMaxDisplayed(normalizeAlertsNumber(maxDisplayed - 1));
+  };
+
+  const onMaxDisplayedAlertsChange = (event: any) => {
+    setMaxDisplayed(normalizeAlertsNumber(Number(event.target.value)));
+  };
+
+  const onMaxDisplayedAlertsPlus = () => {
+    setMaxDisplayed(normalizeAlertsNumber(maxDisplayed + 1));
+  };
+
+  const normalizeAlertsNumber = (value: number) => Math.max(Math.min(value, maxAlerts), minAlerts);
+
+  const alertButtonClasses = 'pf-c-button pf-m-secondary';
+
+  const alertButtonStyle = { marginRight: '8px', marginTop: '8px' };
+
+  const notificationBadge = (
+    <NotificationBadge
+      variant={getUnreadNotificationsNumber() === 0 ? 'read' : 'unread'}
+      onClick={onNotificationBadgeClick}
+      aria-label="Notifications"
+    >
+      <BellIcon />
+    </NotificationBadge>
+  );
+
+  const notificationDrawerActions = [
+    <DropdownItem key="markAllRead" onClick={markAllNotificationsRead} component="button">
+      Mark all read
+    </DropdownItem>,
+    <DropdownItem key="clearAll" onClick={removeAllNotifications} component="button">
+      Clear all
+    </DropdownItem>
+  ];
+
+  const notificationDrawerDropdownItems = (key: React.Key) => [
+    <DropdownItem key="markRead" component="button" onClick={() => markNotificationRead(key)}>
+      Mark as read
+    </DropdownItem>,
+    <DropdownItem key="action" component="button" onClick={() => removeNotification(key)}>
+      Clear
+    </DropdownItem>
+  ];
+
+  const notificationDrawer = (
+    <NotificationDrawer>
+      <NotificationDrawerHeader count={getUnreadNotificationsNumber()} onClose={() => setDrawerExpanded(false)}>
+        <Dropdown
+          onSelect={onDropdownSelect}
+          toggle={
+            <KebabToggle
+              onToggle={isActive => onDropdownToggle('dropdown-toggle-id-0', isActive)}
+              id="dropdown-toggle-id-0"
+            />
+          }
+          isOpen={openDropdownKey === 'dropdown-toggle-id-0'}
+          isPlain
+          dropdownItems={notificationDrawerActions}
+          id="notification-drawer-0"
+          position={DropdownPosition.right}
+        />
+      </NotificationDrawerHeader>
+      <NotificationDrawerBody>
```

Comment:
@mcoker currently the last alert in the NotificationDrawerBody causes and overflow when its kebab toggle is opened, even if there's enough vertical room for the kebab dropdown:

![alert group demo notification drawer overflow](https://user-images.githubusercontent.com/70952936/189918661-67c7bf92-c212-4585-978a-e262a00036af.png)

Either disabled the `overflow-y: auto` property or adding a height property to `pf-c-notification-drawer__body` resolves it (I only added a `height: 100%` to the element to test quickly). I haven't tested with any flex properties , though (since the body element is in the `pf-c-notification-drawer` flex container).

Do you think this would be an update that could be made in Core, or maybe just a style specific to this demo?

Reply:
@thatblindgeye nice catch! That looks like a bug in core to me - https://github.com/patternfly/patternfly/issues/5091